### DISCUSSION TITLE
Roadmaps, Issues etc for ML-KEM and ML-DSA

### DISCUSSION THREAD

**Discussion by sup-main** [open-quantum-safe/liboqs#1837] [General]
Hi Team,

Great work on the liboqs repo.
I wanted to know if there is a list of current issues related to ML-KEM/ML-DSA and a future roadmap for the same that i can refer ? 
e.g. do we have a timeline to implement the changes proposed in 5th PQC conference ?

Thanks.

**Comment by baentsch** [comment]
Thanks for your positive feedback. I'm afraid the contributor base of OQS is a bit too small to adhere to strict timelines or roadmaps (but we welcome contributions :). Regarding your question, could you please be more specific by reference to github issues? We always welcome such hints to give proper priority to our activities. Do you refer to generic topics such as https://github.com/open-quantum-safe/liboqs/issues/1215 that also have a reading on ML-KEM or ML-DSA or some algorithm-specific issues (that possibly might be covered in other repo's issues lists)?

**Comment by sup-main** [comment]
Thanks @baentsch for the reply.

Just wanted to ask/confirm if for future releases from liboqs team is there a feature list/roadmap etc ? or a list of feature that would go in next releases ?
and does next release for ml-kem or ml-dsa depends on the next release from respective pqc author teams ?

**Reply by baentsch** [reply]
The next release has a milestone associated: See https://github.com/open-quantum-safe/liboqs/milestone/25 for the list of issues to be resolved/features to be included. Timeline is not set in stone. And Yes, upstream code updates like ML-KEM, ML-DSA do depend on those teams' timelines.

**Reply by sup-main** [reply]
Thank you @baentsch :)
